src/state/reducers/dayReducer.js

```diff
@@ -0,0 +1,17 @@
+import { day } from '../actions';
+
+const initialState = {
+  currentDate: null,
+  currentDayOfTheWeek: null,
+};
+export const reducer = (state = initialState, action) => {
```

Comment:
Were you able to get the day state working in redux? If not then probably best to delete the dayReducer and dayActions

Reply:
yeah no. I'll delete the dayReducer and dayActions.  Thanks